docs/userguides/accounts.md

```diff
@@ -159,6 +174,24 @@ from ape import accounts
 account = accounts.load("<ALIAS>")
 ```
 
+### Default Sender Support
+
+In order to eliminate the usage of sender in contract calls, you can use `use_sender` context manager.
+
+```python
+with accounts.use_sender(0, 'pass'):
```

Comment:
password should be optional and should use a kwarg in the example

```python
with accounts.use_sender('address', passphrase="pass"):
```

Reply:
The password is not being stored anywhere here; it is being passed in. In some situations this is required, like scripting with keyfile accounts.

That being said, setting the default sender and setting an account to `autosign=True` are two different things and maybe it does not make sense to combine them here.

Reply:
so we dont use passphrase here? keyfile_account needs passphrase

Reply:
Right, but for highlighting best practice, all I meant was this:

```python
import keyring

passphrase = keyring.get_password("ape-scripts", "default-user")
with accounts.use_sender('address', passphrase=password):
```

or even

```python
with accounts.use_sender('address', passphrase=os.environ.get("KEYFILE_PASSPHRASE")):
```

because hardcoded passphrases looks bad from a security perspective, even in documentation.

This is not a blocker for me though, I am only explaining the thought process.

Reply:
ok, got. it